tests/__init__.py

```diff
@@ -2,3 +2,4 @@
 from tests.http_response_test import *
 from tests.encoder_test import *
 from tests.file_test import *
+from tests.serializers.form_part_test import *
```

Comment:
This line is causing problems [in travis](https://travis-ci.org/braintree/braintreehttp_python/jobs/359607711)